### ISSUE TITLE
distribution folder is hard to parse

### ISSUE THREAD

**Issue by tremblap** [flucoma/flucoma-sc#72]
some files in the release-packaging folder are generated, some are editable. If we could keep a constent beaviour at least per folder that would make a lot more sense and make maintenance and upgrades easier to do.

**Comment by tremblap** [comment]
again today, I tried to compile the 'install' folder and it created that architecture - in then end it was faster for me to copy the rendered doc by hand in the package folder (which obviously created a lot of git-diff I didn't commit)
 
<img width="1230" alt="Screenshot 2022-03-24 at 09 13 42" src="https://user-images.githubusercontent.com/10000381/159896730-d314e001-978c-47ff-9ea1-859d32be8320.png">

I think we should make the expected workflow for people to contribute more transparent and straightforward. If I am confused after so many years of hacking around, I reckon a newcomer might be even worse

**Comment by tedmoore** [comment]
@weefuzzy, flagging for us all here that as discussed these folders are changing / going to change, at which point I believe this issue will be resolved.
